pubsub/google/cloud/pubsub_v1/subscriber/_protocol/heartbeater.py

```diff
@@ -0,0 +1,67 @@
+# Copyright 2018, Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import absolute_import
+
+import logging
+import threading
+
+
+_LOGGER = logging.getLogger(__name__)
+_HEARTBEAT_WORKER_NAME = 'Thread-Heartbeater'
+_DEFAULT_PERIOD = 30
```

Comment:
there is no sense of the unit of this from the interface. Not necessary to fix, but a doc comment on the constructor might be helpful.
Also, is there a reason to pick 30 seconds? It is pretty infrequent, but wondering if we can't make it even less frequent?

Reply:
Done.